index.js

```diff
@@ -22,95 +22,112 @@ const playerThreeScore = document.querySelector('.player-three-score');
 let playerOne;
 let playerTwo;
 let playerThree;
+let newGame;
+let question;
+let dailyDouble;
 
 startBtn.addEventListener('click', playAudioIntro);
 
+function submitConditional(answer) {
+  if (playerOne.active) {
+    playerOne.submitAnswer(answer)
+  } else if (playerTwo.active) {
+    playerTwo.submitAnswer(answer)
+  } else if (playerThree.active) {
+    playerThree.submitAnswer(answer)
+  }
+}
+
 submitBtn.addEventListener('click', function(e) {
   e.preventDefault();
   if (hostPrompt.innerText.includes('Player One')) {
-    playerOne = new Player(undefined, undefined, undefined, 1, userInput.value);
+    playerOne = new Player(undefined, undefined, undefined, 1, userInput.value, true);
```

Comment:
Passing in a bunch of undefined values looks more like a bug than a feature here. If these arguments are less important, I'd move them to the end of the parameter & argument list and omit them if they're optional.